pages/a/crate-commoncrawl.html

```diff
@@ -0,0 +1,207 @@
+title: Plugin for Common Crawl
```

Comment:
The title is a bit lame. What would people search for? I guess "Crate Common Crawl plugin" isn't something. I would head more into the direction "Import Data Example Plugin" or "Ingest Custom Formatted Data Plugin". Maybe even something with Copy From. I think those search terms have more values to us.